src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -57,7 +57,17 @@ description: "Learn about the Azure Active Directory Single Sign-On (SSO)."
     - First Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`
 
     - Last Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`
+    
+    - For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
 
    ![Map Identity with provider attributes](azure-map-identity-provider-attributes.png)
+   
+1. Sentry will attempt to authenticate and link your account with Azure. After successful authentication, you'll be redirected to Sentry's SSO configuration page.
```

Comment:
There's a missing connection between this lead in sentence and the bullet points that follow. I've made a suggestion below (and edited to the bullet points to make it work), but you might know a better way to link these. 

```suggestion
1. Sentry will attempt to authenticate and link your account with Azure. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can take the following actions:
```

Reply:
Works for me 👍